tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -544,6 +591,45 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_stuffer_free(&stuffer));
             EXPECT_SUCCESS(s2n_config_free(config));
         }
+
+        /* Test that the client processes extensions.
+         * Specifically, check for the early_data_indication extension. */
+        {
+            const uint32_t expected_max_early_data_size = 17;
+
+            struct s2n_config *config = s2n_config_new();
+            EXPECT_NOT_NULL(config);
+            EXPECT_SUCCESS(s2n_config_set_session_tickets_onoff(config, 1));
+            EXPECT_SUCCESS(s2n_config_set_session_ticket_cb(config, s2n_test_session_ticket_cb, NULL));
+            EXPECT_SUCCESS(s2n_setup_test_ticket_key(config));
+
+            struct s2n_connection *client_conn = s2n_connection_new(S2N_CLIENT);
+            EXPECT_NOT_NULL(client_conn);
+            client_conn->actual_protocol_version = S2N_TLS13;
+            client_conn->secure.cipher_suite = &s2n_tls13_aes_128_gcm_sha256;
+            EXPECT_SUCCESS(s2n_connection_set_config(client_conn, config));
+
+            struct s2n_connection *server_conn = s2n_connection_new(S2N_CLIENT);
```

Comment:
This still needs to be fixed: ```struct s2n_connection *server_conn = s2n_connection_new(S2N_SERVER);```

Reply:
-_- Things went wonky when I rebased. I'll fix it again.